build/Dockerfile

```diff
@@ -43,8 +43,7 @@ COPY utils/scripts/logger.py  utils/scripts/logger.py
 COPY utils/scripts/util.py  utils/scripts/util.py
 RUN  chmod +x run.sh
 COPY requirements.txt setup.py .
-RUN  python3 --version
-RUN  python3 -m venv venv && source venv/bin/activate &&  pip3 install --upgrade pip && venv/bin/pip3 install -r requirements.txt
+RUN  virtualenv -p /usr/bin/python3.8 venv && source venv/bin/activate && pip3 install --upgrade pip && venv/bin/pip3 install -r requirements.txt
```

Comment:
If we don't use the virtualenv we also can delete a new RUN layer

Reply:
We use it. I thought that was obvious.

Reply:
Yes, I see that you use it, that's not what I meant with my comment, let me clarify it. I was just suggesting to not use it to avoid unnecessary extra layers and complexity in the docker image. Is there any chance to avoid it? That's what I asked you in my other comment if there's a reason to use it that I'm missing.

Reply:
Yes, I'm not blocking the merge because of this, but I would like to look at it and see if the Dockerfile can work without the virtualenv. If it does, the changes can be added in a future PR.

Reply:
I'd rather not remove the venv altogether. It doesn't hurt at all and it separates the system python packages from what's used by ods-ci. It's not exactly isolation, but it's much cleaner.